skimage/restoration/tests/test_restoration.py

```diff
@@ -88,17 +91,31 @@ def test_richardson_lucy():
     np.testing.assert_allclose(deconvolved, np.load(path), rtol=1e-3)
 
 
-def test_richardson_lucy_filtered():
+@pytest.mark.parametrize(
```

Comment:
nitpicking here, but I believe
```
@pytest.mark.parametrize('dtype_image', [np.float32, np.float64])
@pytest.mark.parametrize('dtype_psf, [np.float32, np.float64])
def test_richardson_lucy_filtered(dtype_image, dtype_psf):
```
does exactly this, and is a bit more readable

Reply:
Yeah, I got used to the other style from the CuPy test suite, but agree that the above is simpler!